cli/generate.go

```diff
@@ -257,6 +262,14 @@ func runGenCmd(out io.Writer, flags *GenCmdFlags, sedgeAction actions.SedgeActio
 		}
 	}
 
+	// Overwrite feeRecipient and relayURLs for Lido Node
+	if lidoNode {
+		feeRecipient := contracts.FeeRecipient[network]
+		flags.feeRecipient = feeRecipient.FeeRecipientAddress
+
+		flags.relayURLs, _ = mevboostrelaylist.GetRelaysURI(network)
+	}
+
```

Comment:
This is very custom and related to `lidoNode` `nodeSetup` so it is very likely that won't need any refactoring. This is an example of a necessary raw conditional.